conjure-http/src/private/server.rs

```diff
@@ -35,6 +37,22 @@ where
     from_plain(&value, param)
 }
 
+pub fn path_param<T, D>(parts: &request::Parts, param: &str) -> Result<T, Error>
+where
+    D: DecodeParam<T>,
+{
+    let path_params = parts
+        .extensions
+        .get::<PathParams>()
+        .expect("PathParams missing from request");
+    let value = &path_params[param];
```

Comment:
This will panic if `param` is not in the map right? May want to modify `PathParams` to expose the `get` method of the underlying hash map and then map to an error. Unless for some reason `param` is guaranteed to be in here?

Reply:
Yeah, the param is guaranteed to be in there unless the actual server implementation is broken - all of the path params come out of the URI, and we shouldn't be able to route to the handler unless all of the params are present.